alvr/system_info/src/lib.rs

```diff
@@ -145,6 +145,7 @@ pub fn platform() -> Platform {
             ("HTC", "VIVE XR Series", _, _) => Platform::XRElite,
             ("HTC", _, _, _) => Platform::ViveUnknown,
             ("YVR", _, _, _) => Platform::Yvr,
+            ("Play For Dream", _, _, _) => Platform::Yvr,
```

Comment:
Does the PFD have the same exact button layout as the quest? If not you could add another profile. Just going by photos it seems that the grip button is a simple two state button

Reply:
Yeah the side button seems to be digital, though this was also the case with prior YVR headsets I think. But otherwise it's the same and seems to work fine.

Reply:
Sorry, I misread the change. You should actually create a new Platform entry PlayForDreamMR, and then for the controller you can reuse the YVR one